django_ai_assistant/conf.py

```diff
@@ -17,6 +17,7 @@
     "CAN_UPDATE_MESSAGE_FN": "django_ai_assistant.permissions.owns_thread",
     "CAN_DELETE_MESSAGE_FN": "django_ai_assistant.permissions.owns_thread",
     "CAN_RUN_ASSISTANT": "django_ai_assistant.permissions.allow_all",
+    "PRIMARY_KEY_FIELD": "uuid",  # Options: 'auto', 'uuid', 'string'
```

Comment:
I think we don't need an extra setting due to `default_auto_field` at `django_ai_assistant/apps.py`. 

In the example project, you can inherit from `AIAssistantConfig` and set a new `default_auto_field`. See: https://docs.djangoproject.com/en/5.0/ref/applications/#for-application-users

Please document that as a way to change the auto field.